reView/cli.py

```diff
@@ -129,3 +135,259 @@ def unpack_characterizations(
+              f"Valid options are: {TECH_CHOICES}.")
+@click.option('--out_folder', '-o', required=True,
+              type=click.Path(exists=False, dir_okay=True, file_okay=False),
+              help='Path to output folder for maps.')
+@click.option('--boundaries', '-b', required=False,
+              type=click.Path(exists=True, dir_okay=False, file_okay=True),
+              default=DEFAULT_BOUNDARIES,
+              # noqa: E126
+              help=('Path to vector dataset with the boundaries to map. '
+                    'Default is to use state boundaries for CONUS from '
+                    'Natural Earth (1:50m scale), which is suitable for CONUS '
+                    'supply curves. For other region, it is recommended to '
+                    'provide a more appropriate boundaries dataset.'
+                    ))
+@click.option('--dpi', '-d', required=False,
+              default=600,
+              type=click.IntRange(min=0),
+              help='Dots-per-inch (DPI) for output images. Default is 600.')
+def make_maps(
+    supply_curve_csv, tech, out_folder, boundaries, dpi
+):
+    """
+    Generates standardized, presentation-quality maps for the input supply
+    curve, including maps for each of the following attributes:
+    Capacity (capacity), All-in LCOE (total_lcoe), Project LCOE (mean_lcoe),
+    LCOT (lcot), Capacity Density (derived column) [wind only]
+    """
+
+    out_path = Path(out_folder)
+    out_path.mkdir(exist_ok=True, parents=False)
+
+    supply_curve_df = pd.read_csv(supply_curve_csv)
+    supply_curve_gdf = gpd.GeoDataFrame(
+        supply_curve_df,
+        geometry=gpd.points_from_xy(
+            x=supply_curve_df['longitude'], y=supply_curve_df['latitude']
+        ),
+        crs="EPSG:4326"
+    )
+
+    boundaries_gdf = gpd.read_file(boundaries)
+    boundaries_singlepart_gdf = boundaries_gdf.explode(index_parts=True)
+
+    boundaries_dissolved = boundaries_gdf.unary_union
+    background_gdf = gpd.GeoDataFrame(
+        {"geometry": [boundaries_dissolved]},
+        crs=boundaries_gdf.crs
+    ).explode(index_parts=False)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=UserWarning)
+        map_extent = background_gdf.buffer(0.01).total_bounds
+
+    map_vars = {
+        "total_lcoe": {
+            "breaks": [25, 30, 35, 40, 45, 50, 60, 70],
+            "cmap": 'YlGn',
+            "legend_title": "All-in LCOE ($/MWh)"
+        },
+        "mean_lcoe": {
+            "breaks": [25, 30, 35, 40, 45, 50, 60, 70],
+            "cmap": 'YlGn',
+            "legend_title": "Project LCOE ($/MWh)"
+        },
+        "lcot": {
+            "breaks": [5, 10, 15, 20, 25, 30, 35, 40, 50],
+            "cmap": 'YlGn',
+            "legend_title": "LCOT ($/MWh)",
+        }
+    }
+    if tech == "solar":
+        map_vars.update({
+            "capacity": {
+                "breaks": [100, 500, 1000, 2000, 3000, 4000],
```

Comment:
So it seems that the purpose of the tech choice here is to set the breaks in the capacity? Is there any way to make it even more generalizable than this? For example, I would love to make some goethermal supply curves using this functionality. but that wouldn't be currently allowed unless I "spoof" the supply curve to be wind or solar. 

I assume these may have been derived by visual inspection, but couldn't we make a general case that tried to select intervals based on equal binning? I'm pretty sure numpy has functionality for something like this. (If you thin this is dowable let me know and I can dig up the numpy function)

Reply:
Yea, I quite like the idea of a `breaks-style` argument. the more flexible, the better IMO, but even just the three options (`wind`, `solar`, `auto`) would be great.

This isn't urgent at all though, so if you are eager to merge this PR I would say do that and this change can come later down the line.